.github/workflows/nightly.yaml

```
@@ -0,0 +1,27 @@
+name: Nightly
+
+on:
+  workflow_dispatch:
+  # Instead of running every night we run
+  # Once for every commit in main as the traffic on this repo
+  # is lower than facebook/react-native.
+  push:
+    branches:
+      - main
+
+jobs:
+  publish_template_as_nightly:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.1.1
+      - name: Setup node.js
+        uses: actions/setup-node@v4.0.0
+        with:
+          node-version: 18
+      - name: Update versions to nightly
+        run: node updateTemplateVersion.js nightly
+      - name: Publish NPM as Nightly
+        run: npm publish --dry-run --tag nightly
```

are you sure about the `--dry-run` parameter? My gut feeling is that with the `--dry-run` parameter set we will not actually publish the template. 🤔 
```suggestion
        run: npm publish --tag nightly
```

---

I'd like to have a couple of runs with `--dry-run` and will remove it afterwards in a separate PR